lib/index.js

```diff
@@ -176,24 +176,10 @@ api.createAuthzHeader = (
   params.headers = includeHeaders.join(' ');
   params.signature = signature;
   if(created !== undefined) {
-    if(created instanceof Date) {
-      params.created = created.getTime();
-    } else if(Number.isInteger(created)) {
-      params.created = created;
-    } else {
-      throw new TypeError(
-        '"created" must be a UNIX timestamp or JavaScript Date.');
-    }
```

Comment:
We still want these checks -- we can call `validateDate` if that function no longer checks ranges.

Reply:
I have 2 different commits related to this:

https://github.com/digitalbazaar/http-signature-header/pull/17/commits/c25f7bdacb8bde4de0697e2ee2a1f7ad0f29d319

and then

https://github.com/digitalbazaar/http-signature-header/pull/17/commits/b095befd3619b794f34d3818ea9ddaa44552891d